backend/src/main/java/com/redhat/cloud/notifications/recipients/itservice/pojo/request/AccountRelationship.java

```diff
@@ -0,0 +1,37 @@
+package com.redhat.cloud.notifications.recipients.itservice.pojo.request;
+
+import com.fasterxml.jackson.annotation.JsonInclude;
+import com.fasterxml.jackson.annotation.JsonProperty;
+import com.fasterxml.jackson.annotation.JsonPropertyOrder;
+
+import java.util.List;
+
+@JsonInclude(JsonInclude.Include.NON_NULL)
+@JsonPropertyOrder({
+        "allOf",
+        "by"
+})
+public class AccountRelationship {
+
+    @JsonProperty("allOf")
```

Comment:
General comment: `@JsonProperty` is not useful if its `value` field is exactly the same than the Java field name. Most (maybe all) `@JsonProperty` annotations can be removed from this PR.

Reply:
I actually like to have properties like this (even in JPA Entities) because the code gets independent from the table names/json attribute names and we can refactor the code without any side effects.

Reply:
> I actually like to have properties like this (even in JPA Entities) because the code gets independent from the table names/json attribute names and we can refactor the code without any side effects.

While this makes perfect sense, it is still debatable. 😃 

I never code for an hypotetical future situation because most of the time that situation will never happen. Coding for the present usually makes the code lighter and easier to maintain.

Reply:
me neither normally. I am usually taking the pragmatic way. But in this case I like being "refactoring safe".

Reply:
we can remove it as well.. i can live with both :)